pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
```

Comment:
BaselineVersion is used to determine the minimum version used for all packages in the repo.

By default we use the AssemblyVersion in packageVersion map to understand what version of a package to reference.  Then we'll raise the package version of the reference so that its at least the baseline version.